Extension/src/LanguageServer/ui.ts

```diff
@@ -307,24 +308,17 @@ export class OldUI implements UI {
         }
     }
 
-    private compilerTimout?: NodeJS.Timeout;
-    public async showCompilerStatusIcon(show: boolean): Promise<void> {
-        if (!telemetry.showStatusBarIntelliSenseIndicator()) {
+    public async showConfigureIntelliSenseStatusIcon(show: boolean, client?: Client): Promise<void> {
```

Comment:
For clarity, I would think "button" was a better description of what we're using over "StatusIcon"